src/extension/prompts/node/agent/summarizedConversationHistory.tsx

```diff
@@ -194,7 +194,8 @@ export class AgentSummarizationPrompt extends PromptElement<ConversationHistoryS
 
 					This summary should serve as a comprehensive handoff document that enables seamless continuation of all active work streams while preserving the full technical and contextual richness of the original conversation.<br />
 				</SystemMessage>
-				{/* Dynamic history components - not cached for optimal cache efficiency */}
+				<cacheBreakpoint type={CacheType} />
```

Comment:
Ah, I think I didn't explain well enough- I'm hoping to hit the prompt cache from the user's own previous normal chat requests. This could hit a cache from the user's previous summarization requests, but I don't think that will happen that often, and doesn't include most of the content anyway